augur/filter.py

```diff
@@ -25,13 +25,20 @@ def read_vcf(compressed, input_file):
                 return seq_keep, all_seq
 
 
-def write_vcf(compressed, input_file, output_file, dropped_samps):
-    #Read in/write out according to file ending
-    inCall = "--gzvcf" if compressed else "--vcf"
-    outCall = "| gzip -c" if output_file.lower().endswith('.gz') else ""
+def write_vcf(input_filename, output_filename, dropped_samps):
```

Comment:
Same concern as #478 about changing the API.  If this is only used internally it is probably ok, but if they have any users this will cause code breakage.